ember/app/ui/cases/index/controller.js

```diff
@@ -105,17 +111,39 @@ export default class CasesIndexController extends TableController {
       return [];
     }
 
-    await Promise.resolve();
-
     try {
-      return (
+      const accesses = (
         await this.store.query("case-access", {
           filter: { idpIds: idpIds.join(",") },
         })
       ).map((access) => access.get("caseId"));
+
+      // filtering against an empty list is treated as a no-op, therefore return a dummy value
+      return accesses.length
+        ? accesses
+        : ["00000000-0000-0000-0000-000000000000"];
```

Comment:
Reverts removal of fix in https://github.com/adfinis/mySAGW/commit/6f0982a1d85dfe26df22208f29b961bc05b2e2a0